### PR TITLE
Update plan-for-the-multilingual-user-interface.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed the invalid link with https://docs.microsoft.com/en-us/sharepoint/add-and-edit-user-profile-properties

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/sites/plan-for-the-multilingual-user-interface.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ By default, when a new site is created, it is created in the default language of
...
+1. If the User Profile service application is started on the SharePoint Server farm, the language preferences stored in the user profile are used. For information about how to add a list of languages to user profile settings in SharePoint Server, see [Add, edit, or delete custom properties in SharePoint Server user profiles](../administration/add-edit-or-delete-custom-properties-for-a-user-profile.md). For information about how to add a list of languages to user profile settings in SharePoint Online Administration Center, see [Add and edit user profile properties](https://docs.microsoft.com/en-us/sharepoint/add-and-edit-user-profile-properties).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/sites/plan-for-the-multilingual-user-interface.md]:
```suggestion
1. If the User Profile service application is started on the SharePoint Server farm, the language preferences stored in the user profile are used. For information about how to add a list of languages to user profile settings in SharePoint Server, see [Add, edit, or delete custom properties in SharePoint Server user profiles](https://docs.microsoft.com/en-us/sharepoint/administration/add-edit-or-delete-custom-properties-for-a-user-profile). For information about how to add a list of languages to user profile settings in SharePoint Online Administration Center, see [Add and edit user profile properties](https://docs.microsoft.com/en-us/sharepoint/add-and-edit-user-profile-properties).
```

File: SharePoint/SharePointServer/sites/plan-for-the-multilingual-user-interface.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ By default, when a new site is created, it is created in the default language of
...
+1. If the User Profile service application is started on the SharePoint Server farm, the language preferences stored in the user profile are used. For information about how to add a list of languages to user profile settings in SharePoint Server, see [Add, edit, or delete custom properties in SharePoint Server user profiles](https://docs.microsoft.com/en-us/sharepoint/administration/add-edit-or-delete-custom-properties-for-a-user-profile). For information about how to add a list of languages to user profile settings in SharePoint Online Administration Center, see [Add and edit user profile properties](https://docs.microsoft.com/en-us/sharepoint/add-and-edit-user-profile-properties).
```
Comment (nenonix) [inline SharePoint/SharePointServer/sites/plan-for-the-multilingual-user-interface.md]:
```suggestion
1. If the User Profile service application is started on the SharePoint Server farm, the language preferences stored in the user profile are used. For information about how to add a list of languages to user profile settings in SharePoint Server, see [Add, edit, or delete custom properties in SharePoint Server user profiles](https://docs.microsoft.com/sharepoint/administration/add-edit-or-delete-custom-properties-for-a-user-profile). For information about how to add a list of languages to user profile settings in SharePoint Online Administration Center, see [Add and edit user profile properties](https://docs.microsoft.com/sharepoint/add-and-edit-user-profile-properties).
```

**Comment by scanum** [comment]
@bentoncity Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**